lib/galaxy_test/api/test_datasets.py

```diff
@@ -462,6 +464,14 @@ def test_delete_batch(self):
             {"id": dataset_map[0], "src": "hda"},
             {"id": dataset_map[2], "src": "hda"},
         ]
+
+        # Ensure dataset files exists on disk
+        dataset_files_to_purge = []
+        for purged_source_id in expected_purged_source_ids:
+            dataset_file = self.dataset_populator.get_underlying_dataset_on_disk(purged_source_id["id"])
+            assert self.dataset_populator.file_exists_on_disk(dataset_file)
+            dataset_files_to_purge.append(dataset_file)
+
```

Comment:
API tests should pass irrespective of the server config and should not require disk access to the object store. Can you create an integration test for this ?

Reply:
Sure!